Makefile

```diff
@@ -59,9 +61,16 @@ formatter:
 	poetry run black rasa tests
 
 lint:
-	poetry run flake8 rasa tests
+    # Ignore docstring errors when running on the entire project
+	poetry run flake8 rasa tests --extend-ignore D
 	poetry run black --check rasa tests
 
+branch ?= master
```

Comment:
this checks for a mandatory `branch` parameter? neat, didn't know about this

Reply:
No, it provides a default it isn't provided. Added a comment.